src/script/components/InputBar/InputBar.tsx

```diff
@@ -861,6 +887,29 @@ const InputBar = ({
       id="conversation-input-bar"
       className={cx('conversation-input-bar', {'is-right-panel-open': isRightSidebarOpen})}
     >
+      {activeTypingUsers.length > 0 && (
+        <div className="conversation-input-bar-typing-users-container">
+          <div className="conversation-input-bar-typing-users-avatars-container">
+            {activeTypingUsers.slice(0, 3).map((user, index) => (
+              <Avatar
+                key={user.id}
+                className={cx('cursor-default', {'conversation-input-avatar-with-margin-left': index > 0})}
+                participant={user}
+                avatarSize={AVATAR_SIZE.X_SMALL}
+              />
+            ))}
+          </div>
+          {activeTypingUsers.length === 1 &&
+            t('tooltipConversationInputOneUserTyping' as StringIdentifer, {user1: activeTypingUsers[0].name()})}
+          {activeTypingUsers.length === 2 &&
+            t('tooltipConversationInputTwoUserTyping' as StringIdentifer, {
+              user1: activeTypingUsers[0].name(),
+              user2: activeTypingUsers[1].name(),
+            })}
+          {activeTypingUsers.length > 2 &&
+            t('tooltipConversationInputMoreThanTwoUserTyping' as StringIdentifer, {user1: activeTypingUsers[0].name()})}
+        </div>
+      )}
```

Comment:
Let's extract this to a component that has it's own styles/render function/tests/states... :)